tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -111,70 +110,106 @@ def supported_precision_modes():
+        TRT network and engine at run time. i.e. Since TensorRT version < 6.0
+        does not support dynamic dimensions other than the batch dimension,
+        when the TensorFlow graph has a non-batch dimension of dynamic size,
+        we would need to enable this option. This option should be set to True
+        in TF 2.0.
+      maximum_cached_engines: max number of cached TRT engines for dynamic TRT
+        ops. Created TRT engines for a dynamic dimension are cached. This is
+        the maximum number of engines that can be cached. If the number of
+        cached engines is already at max but none of them supports the input
+        shapes, the TRTEngineOp will fall back to run the original TF subgraph
+        that corresponds to the TRTEngineOp.
+      use_calibration: this argument is ignored if precision_mode is not INT8.
+        If set to True, a calibration graph will be created to calibrate the
+        missing ranges. The calibration graph must be converted to an inference
+        graph by running calibration with calibrate(). If set to False,
+        quantization nodes will be expected for every tensor in the graph
+        (exlcuding those which will be fused). If a range is missing, an error
+        will occur. Please note that accuracy may be negatively affected if
+        there is a mismatch between which tensors TRT quantizes and which
+        tensors were trained with fake quantization.
+      max_batch_size: max size for the input batch. This parameter is only
+        effective when is_dynamic_op=False which is not supported in TF 2.0.
+    """
+    super(TrtConversionParams, self).__setattr__(
+        "rewriter_config_template", rewriter_config_template)
+    super(TrtConversionParams, self).__setattr__(
+        "max_workspace_size_bytes", max_workspace_size_bytes)
+    super(TrtConversionParams, self).__setattr__(
+        "precision_mode", precision_mode)
+    super(TrtConversionParams, self).__setattr__(
+        "minimum_segment_size", minimum_segment_size)
+    super(TrtConversionParams, self).__setattr__(
+        "is_dynamic_op", is_dynamic_op)
+    super(TrtConversionParams, self).__setattr__(
+        "maximum_cached_engines", maximum_cached_engines)
+    super(TrtConversionParams, self).__setattr__(
+        "use_calibration", use_calibration)
+    super(TrtConversionParams, self).__setattr__(
+        "max_batch_size", max_batch_size)
+
+  def __setattr__(self, key, value):
+    """Override __setattr__ to make class immutable."""
+    raise AttributeError(
+        "{} is immutable and thus cannot change its attributes".format(
+            self.__class__))
+
+  def _replace(self,
+               rewriter_config_template=None,
+               max_workspace_size_bytes=None,
+               precision_mode=None,
+               minimum_segment_size=None,
+               is_dynamic_op=None,
+               maximum_cached_engines=None,
```

Comment:
`Args section is missing, does not match arg list or is badly indented.

Expected: rewriter_config_template, max_workspace_size_bytes, precision_mode, minimum_segment_size, is_dynamic_op, maximum_cached_engines, use_calibration, max_batch_size

Found: 

Missing: rewriter_config_template, max_workspace_size_bytes, precision_mode, minimum_segment_size, is_dynamic_op, maximum_cached_engines, use_calibration, max_batch_size

Unexpected: `